### DISCUSSION TITLE
Latest Version  3.13.0-1 Tremulant activation

### DISCUSSION THREAD

**Discussion by jefferycrowley** [GrandOrgue/grandorgue#1723] [Help]
Hello all good GO People!

This past week after installing the latest GO version for Windows - 3.13.0-1;  first of all bravo for ASIO inclusion finally.  

I have my own version of Litomysl from Sonus Paradisi, which is basically with only three changes.   On the Bombardwerk, I gave the Trompet imperial 32' a full compass and transmitted it to the pedal completely re-voiced it in Grand Orgue so it would work with both the Bombardwerk  other horizontal reeds AND the Pedalwerk stops.

I added a tremulant on the Hauptwerk.

I created a completely new independent Cornet VI for the Bombardwerk with 100% material from the rest of the organ and used these samples as duplicates in their own folder. (That way I did not have to resample them or loop them!).   This replaces the very odd Cornet VI that came with the instrument has zero breaks and is a pure joy to play and it is BIG!

I created all new Graphics from scratch that mimic the real instruments console - added two new side panels for the Stop Jambs.  I also used the Hauptwerk 2 files supplied on my disc that had all of the mechanical sounds such as trackers, blower, tremulant noise, stop action but did not employ them in Hauptwerk.   

So far I have only the tracker noise working (Thanks Lars Palo!)  I was unable to record anything using this GO creation because the latency was too great, but now with ASIO there is no problem AND it sounds fantastic (as long as I truncate the reverb to 2500 ms).  I have been unable to use this version in Hauptwerk because there is something wrong with the Wind Module and even though I turned it off for this sample set in HW it still sounds as if I am continually robbing the wind supply of air.  So a lot of wheezing, gasping and generally not working correctly.

My actual question is last week with the new version of GO installed - I had been playing the sample set for most of the morning,  I left the house to do a lot of chore-work and came back several hours later and still had the Litomysl sample-set loaded in GO.  When I began to play the Hauptwerk manual seemed to have a tremulant in the ON State.  It effected every Hauptwerk Stop, BUT, it was not the Hauptwerk Tremulant.   None of the other tremulants were in the "ON" state and even when I drew the Hauptwerk Tremulant it was different than the one that was running in the background.    

I fixed it by closing the sample-set and reloading it in GO.  This has since happened with many of my Grand Orgue Sample-sets.  Both those available in GO and also instruments that I have completely re-done the graphics on including early HW sample-sets.

So the _modus operandi_, is leaving the sample set up and running in Grand Orgue and not playing it for quite a while.

**Comment by oleg68** [comment]
1. How often does it occur? Every day or only once?
2. Does it affect only with the first manual stops or also with another manual stops?
3. Does it occur with the demo organ?
4. If the problem is reproducible, could you record an audio (Panel->Recorder, Audio Recorder, REC) of one note of one stop just after loading the sample set and then the same note after the ossue occur?
